packages/cli/skills/pikku-auth-js/SKILL.md

```
@@ -1,106 +1,318 @@
-### Wiring Auth Routes
+---
+
+## Login / Logout from the Frontend
+
+Auth.js handles these via its standard endpoints. With `basePath: '/auth'`:
+
+### Login
+
+`POST /auth/callback/credentials` with a `application/x-www-form-urlencoded` body:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)